civis/ml/tests/test_helper.py

```diff
@@ -0,0 +1,25 @@
+import mock
+import pytest
+
+from civis.response import Response
+from civis.ml import list_models
+
+
+def test_list_models_bad_job_type():
+    with pytest.raises(ValueError):
+        list_models(job_type="fake")
+
+
+def test_list_models():
+    resp = [Response({'id': 2834, 'name': 'RFC model'})]
+    m_client = mock.Mock()
```

Comment:
This is a good place to use the (poorly documented) `create_client_mock` function, https://github.com/civisanalytics/civis-python/blob/v1.10.0/civis/tests/mocks.py#L15 .